src/jetson/encryptor.py

```diff
@@ -3,11 +3,11 @@
 from typing import List, Tuple
 
 class Encryptor(object):
```

Comment:
I think this class is backwards. Why do we need to re-encapsulate `AESEncryptor` with `Encryptor`? I think it would be much better for `AESEncryptor` to make all encryptors just implement `encryptFace` and `encryptFrame` themselves. This will then resolve the weird import statement `from src.jetson.AES import Encryption as AESEncryptor` which is very misleading.